### PR TITLE
Support custom model URL

### PR DESCRIPTION
**Pull request by universeroc**
I want to load a local model rather than tfhub.dev to improve the loading performance. But I can not load model except from tfhub.dev, so I make this patch to make it work, if there's another way to make it let me know, thank you in advance :)

### CODE REVIEW

**Review by ahmedsabie — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-models/1027)* status: 0 of 1 approvals obtained (waiting on @universeroc)

___
*[`toxicity/src/index.ts` line 37 at r1](https://reviewable.io/reviews/tensorflow/tfjs-models/1027#-NAzJ7Oo0i-x5GBqDkcA:-NAzJ7Op6w1jPkZcVQEy:b-1uqyp5) ([raw file](https://github.com/tensorflow/tfjs-models/blob/7db25b43892bc8143bb3d6b58c017c3b174c3235/toxicity/src/index.ts#L37)):*
> ```XML
>  * `identity_attack` | `insult` | `threat` | `sexual_explicit` | `obscene`.
>  * Defaults to all labels.
>  * @param modelURL Load a model from URL rather than tfhub.dev
> ```

Rephrase it to "URL to load model from. Defaults to tfhub.dev.
___
*[`toxicity/src/index.ts` line 61 at r1](https://reviewable.io/reviews/tensorflow/tfjs-models/1027#-NAzJ3BG9hSqPt1nOVc6:-NAzJ3BH7IOYwJszKCW6:bdpg86a) ([raw file](https://github.com/tensorflow/tfjs-models/blob/7db25b43892bc8143bb3d6b58c017c3b174c3235/toxicity/src/index.ts#L61)):*
> ```XML
>   }
> 
>   setModelURL(url: string) {
> ```

Please make this private

**Review by ahmedsabie — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-models/1027)* status: 0 of 1 approvals obtained (waiting on @universeroc)

___
*[`toxicity/src/index.ts` line 37 at r1](https://reviewable.io/reviews/tensorflow/tfjs-models/1027#-NAzJ7Oo0i-x5GBqDkcA:-NCurvmq5tiNlG5SW1RS:bas9s0f) ([raw file](https://github.com/tensorflow/tfjs-models/blob/7db25b43892bc8143bb3d6b58c017c3b174c3235/toxicity/src/index.ts#L37)):*
<details><summary><i>Previously, ahmedsabie wrote…</i></summary><blockquote>

Rephrase it to "URL to load model from. Defaults to tfhub.dev.
</blockquote></details>

Sorry I should have been clearer, the argument name should still stay as modelURL, so the whole line should be:



_Code snippet:_
```XML
* @param modelURL. URL to load model from. Defaults to tfhub.dev.
```
___
*[`toxicity/src/index.ts` line 38 at r2](https://reviewable.io/reviews/tensorflow/tfjs-models/1027#-NCus9xZAYA6WlxaREjd:-NCus9xZAYA6WlxaREje:bbg03tc) ([raw file](https://github.com/tensorflow/tfjs-models/blob/b08beb3769c96d256ab270c29117054077379384/toxicity/src/index.ts#L38)):*
> ```XML
>  * Defaults to all labels.
>  * @param URL to load model from. Defaults to tfhub.dev
>  * **Default** is tfhub.dev*.
> ```

Remove the duplicate bold part
___
*[`toxicity/src/index.ts` line 41 at r2](https://reviewable.io/reviews/tensorflow/tfjs-models/1027#-NCusBfl8MTyqbvhUwnU:-NCusBfl8MTyqbvhUwnV:b-jqynn5) ([raw file](https://github.com/tensorflow/tfjs-models/blob/b08beb3769c96d256ab270c29117054077379384/toxicity/src/index.ts#L41)):*
> ```XML
>  */
> export async function load(threshold: number, toxicityLabels: string[],
>     URL: 'https://tfhub.dev/tensorflow/tfjs-model/toxicity/1/default/1') {
> ```

rename back to modelURL

**Comment by universeroc** [comment]
@ahmedsabie setModelURL should be set private as in function `load` call this `setModelURL` There's no friend keyword like C++ so I remove the private keyword in this commit.
